base/docs/helpdb/Base.jl

```diff
@@ -1767,16 +1767,19 @@ deserialize
 """
     length(collection) -> Integer
 
-For ordered, indexable collections, returns the maximum index `i` for which `getindex(collection, i)`
-is valid.
-For unordered collections, returns the number of elements.
+Return the number of elements in collection.
```

Comment:
Well, that's how it's done elsewhere, e.g. for `endof`. The "collection" mentioned in this docstring can always be interpreted as "the argument `collection`". `A` evokes an array rather than a general collection.

Though we could certainly change our conventions, but I'd rather do this in a separate PR.